src/frontend/components/SøknadsSteg/Dokumentasjon/LastOppVedlegg.tsx

```diff
@@ -29,7 +30,7 @@ const Container = styled.div`
 
 const LastOppVedlegg: React.FC<Props> = ({ dokumentasjon, vedleggNr }) => {
     const { søknad, settSøknad } = useApp();
-    const { formatMessage } = useIntl();
+    const intl = useIntl();
```

Comment:
Hvorfor er dette bedre?

Reply:
Var ikke bedre, vi bare trengte intl-objektet for å sendes inn i funksjon lenger ned.